zxing_standalone/src/main/java/com/google/zxing/client/android/DecodeHandler.java

```diff
@@ -81,9 +83,22 @@ private void decode(byte[] data, int width, int height) {
         rawResult = multiFormatReader.decodeWithState(bitmap);
       } catch (ReaderException re) {
         // continue
-      } finally {
+      } 
+      finally {
         multiFormatReader.reset();
       }
+
```

Comment:
@tarun0 hi, I just push the changes in a only one commit how you say to maintain the repository fine. ;)

Reply:
Thanks for the contribution 👍